snippets/functions/third-party/awsKinesis.js

```diff
@@ -0,0 +1,26 @@
+exports = async function () {
+  const {
+    KinesisClient,
+    DescribeStreamCommand,
+  } = require("@aws-sdk/client-kinesis");
+  const kinesisClient = new KinesisClient({
+    credentials: {
+      accessKeyId: context.values.get("awsAccessKeyID"),
+      secretAccessKey: context.values.get("awsSecretAccessKey"),
+    },
+    region: "us-east-1",
+    maxAttempts: 5,
+  });
+  const params = {
+    StreamName: "myStreamName",
+  };
+
+  // see https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/kinesis/ for more commands
+  const describeStreamCommand = new DescribeStreamCommand(params);
```

Comment:
Very minor but it's to stay consistent with the other snippets.

```suggestion
  const describeStreamCommand = new DescribeStreamCommand({
    StreamName: "myStreamName",
  });
```